rezerwacje-front/src/views/Users/components/UsersForm/UsersForm.js

```diff
@@ -0,0 +1,276 @@
+                required
+                variant="outlined"
+                value={values.email}
+              />
+              <p className="error">{errors.email}</p>
+            </Grid>
+            <Grid
+              item
+              md={6}
+              xs={12}
+            >
+              <TextField
+                fullWidth
+                label="Numer telefonu"
+                margin="dense"
+                name="phone"
+                onChange={handleChange}
+                type="phone"
+                value={values.phone}
+                variant="outlined"
+              />
+              <p className="error">{errors.phone}</p>
+            </Grid>
+            <Grid
+              item
+              md={12}
+              xs={12}
+            >
+            <Typography
+               gutterBottom
+               variant="h6">
+                 Dane logowania
+             </Typography>
+             </Grid>
+            <Grid
+              item
+              md={12}
+              xs={12}
+            >
+              <TextField
+                fullWidth
+                label="Login"
+                margin="dense"
+                name="login"
+                onChange={handleChange}
+                required
+                value={values.login}
+                variant="outlined"
+              />
+              <p className="error">{errors.login}</p>
+            </Grid>
+            <Grid
+              item
+              md={6}
+              xs={12}
+            >
+              <TextField
+                fullWidth
+                label="Hasło"
+                margin="dense"
+                name="password"
+                type="password"
+                onChange={handleChange}
+                required
+                value={values.password}
+                variant="outlined"
+              />
+            <p className="error">{errors.password}</p>
+            </Grid>
+            <Grid
+              item
+              md={6}
+              xs={12}
+            >
+           <TextField
+                fullWidth
+                label="Powtórz hasło"
+                margin="dense"
+                name="confirmPassword"
+                type="password"
+                onChange={handleChange}
+                required
+                variant="outlined"
+              />
+           <p className="error">{errors.confirmPassword}</p>
+           </Grid>
+           <Grid
+              item
+              md={12}
+              xs={12}
+            >
+            <Typography
+               gutterBottom
+               variant="h6">
+                 Uprawnienia
+             </Typography>
+             </Grid>
+          </Grid>
+          <Grid
+          item
+          md={12}
+          xs={12}>
+          <RadioGroup name="role">
+               <FormControlLabel value="ADMINISTRATOR" control={<Radio color='primary'/>} label="Administrator" onChange={handleChange}/>
```

Comment:
This is strange, when you select buttons the selection dot appears a bit off to the left

Reply:
https://github.com/niewidoczni-akademicy/rezerwacje-sal/blob/019e6368918bc5068d91037cc0e2adcf72233793/rezerwacje-front/src/views/Rooms/components/RoomsForm/RoomsForm.scss
It's because in this file there are styles for standard elements, which affect every element on the page, not just the elements the creator wanted to affect. It's a big issue that needs to be fixed, maybe with classes, or with styled-components